inscriptions/ISic000708.xml

```diff
@@ -190,11 +195,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/>RO<gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
characters which cannot be given any sense are presented in capitals as here, but also placed inside an `<orig>` element